service/controller/appcatalog/v1/resource/index/resource.go

```diff
@@ -19,20 +20,24 @@ type Config struct {
 	// Dependencies.
 	K8sClient kubernetes.Interface
 	Logger    micrologger.Logger
+
+	// Settings.
```

Comment:
The settings need to be ordered in the resource. Its nitpicky but the code gets untidy quickly when stuff isn't ordered.

Reply:
Sure, will keep in mind.